go.sum

```diff
@@ -0,0 +1,43 @@
+github.com/davecgh/go-spew v1.1.0 h1:ZDRjVQ15GmhC3fiQ8ni8+OwkZQO4DARzQgrnXU1Liz8=
```

Comment:
Could you remove the go.sum file? Since this is generated we don't save this in the repository.

Reply:
removed, is it worth adding this to the gitignore or would this mess with releases?